config/constants.php

```diff
@@ -0,0 +1,8 @@
+<?php
+	return [
+		'LIMIT_NEWS_INDEX' => '2',
```

Comment:
no need this `const` the Eloquent have the `perPage` property, default is 15, If you want to custom, just define in the model class